contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol

```diff
@@ -203,13 +203,6 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
       revert InsufficientBalance();
     }
 
-    bytes32 requestId = _computeRequestId(
```

Comment:
Let's do contract changes in a separate PR for easier tracking/review.

Reply:
Unless you feel strongly about this, I'd prefer to keep this entire change in a single PR since the changes are dependent on each other.